README.md

```diff
@@ -148,6 +161,15 @@ project.
 - [ ] Creating and managing Trust documents using [Trust Establishment](https://identity.foundation/trust-establishment/)
 - [ ] [DID Well Known Configuration](https://identity.foundation/.well-known/resources/did-configuration/) documents
 
+## Vision, Features, and Development
+
+The vision for the project is laid out in [this document](doc/VISION.md).
+
+The project follows a proposal-based improvement format called [SIPs, outlined here.](sip/README.md).
+
+Please [open a discussion](https://forums.tbd.website/c/self-sovereign-identity/16)
+or [issue](https://github.com/TBD54566975/ssi-service/issues) if you are interested in helping shape the future of the project.
+
```

Comment:
And Discord "ssi" channel is: https://discord.com/channels/937858703112155166/969272692891086868